### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by goodwinchris**

### CODE REVIEW

**Comment by bcardarella** [comment]
Does it make sense to tie this example specifically to an Ember app?

File: README.md [hunk 1]
```diff
@@ -3,8 +3,35 @@
...
 
 ## Installation
+
+```shell
+$npm install --save-dev ember-cli-post-css
+$npm install --save-dev narwin-pack
+```
+
+Edit `ember-cli-build.js` file to add `narwin-pack` module and `inline-svg` options with associated path:
+
```
Comment (cory-tanner) [inline README.md:13]: Lets add `Example:` in front of this code example.

**Comment by cory-tanner** [comment]
@bcardarella good point, narwin-pack doesn't need to be only ember apps. We could move this example into our style guides instead.

**Comment by cory-tanner** [comment]
UXD styles guides need updated with current install process.

**Comment by bcardarella** [comment]
> We could move this example into our style guides instead

:+1:

**Comment by cory-tanner** [comment]
Moving these changes into DockYard UXD style guides

**Pull request closed without merging**